props/saber_BC_buttons.h

```diff
@@ -364,14 +1120,33 @@ Turn OFF without postoff - Turn OFF while pointing up.
 #define THRUST_GESTURE
 #endif
 
+#ifndef BC_MAIN_BLADE
+#define BC_MAIN_BLADE 1
+#endif
+
+#ifndef BC_SECOND_BLADE
+#define BC_SECOND_BLADE 2
+#endif
+
+#ifdef BC_DUAL_BLADES
+#if BC_MAIN_BLADE == BC_SECOND_BLADE
+#error BC_MAIN_BLADE and BC_SECOND_BLADE cannot be the same
+#endif
+#endif
+
 #if defined(NO_BLADE_NO_GEST_ONOFF) && !defined(BLADE_DETECT_PIN)
-#error Using NO_BLADE_NO_GEST_ONOFF requires a BLADE_DETECT_PIN to be defined 
+#error Using NO_BLADE_NO_GEST_ONOFF requires a BLADE_DETECT_PIN to be defined
```

Comment:
This might be a TODO for later, but doesn't NO_BLADE_NO_GEST_ONOFF work with using continuous blade detect and NO_BLADE_ID_RANGE?

Reply:
It could. I added BLADE_ID_SCAN_MILLIS to the error, is it ok to do the 2 small changes I made to prop_base?
- Unguarding this global variable
```cpp
#ifdef BLADE_DETECT_PIN
  bool blade_detected_ = false;
#endif
```
- and adding this to ScanBladeIdNow()
`      blade_detected_ = (current_config->ohm < NO_BLADE);`

Reply:
replace blade_detected_ with blade_present() in your prop instead.

Reply:
omg I didn't even see that and I was right above it too.
Thanks, ok done.